Classes/Information/EventInformation.php

```diff
@@ -55,4 +70,20 @@ public function getCreatorInformation(): CreatorInformation
     {
         return $this->creatorInformation;
     }
+
+    public function setExtensionInformation(ExtensionInformation $extensionInformation): void
```

Comment:
I don't understand all these setters, as these classes are already injected in constructor

Reply:
In the command we will create an empty Information object, then step by step ask all questions and use the setters to set the answers. 

This way we can use previous answers to generate default values, possible options or autocomplete solutions.